py/py_sparkling/ml/util.py

```diff
@@ -26,6 +26,12 @@ def getDoubleArrayFromIntArray(array):
     else:
         return list(map(float, array))
 
+def getDoubleArrayArrayFromIntArrayArray(array):
```

Comment:
moved to ai.h2o.sparkling.utils as new methods should be introduced in the new package